client/static/js/layout.js

```diff
@@ -0,0 +1,61 @@
+const { sendPost, getPost } = require('./api.js')
+const { encode } = require('html-entities');
+
+window.addEventListener('hashchange', updateContent);
+window.addEventListener('load', updateContent);
+
+
+function updateContent() {
+    const root = document.getElementById('root')
+    root.innerHTML = ''
+    const id = window.location.hash.substring(1)
+    id.length > 0 ? renderPost(id) : renderForm()
+}
+
+async function renderPost(id){
+    const root = document.getElementById('root')
+    const postData = await getNewPost(id)
+    root.innerHTML = `
+    <div class="post-cont">
+    <h2>${encode(postData.title)}</h2>
+    <h3 class="post" id="name">${encode(postData.pseudonym)}</h3><span> ● ${prettyDate(postData.date)}</span>
+    <p>${encode(postData.content)}<p>
+    </div>`
+}
+
+function prettyDate(date) {
+    const jsDate = new Date(Date.parse(date))
+    const day = jsDate.getDate()	
+    const month = jsDate.getMonth() + 1
+    const year = jsDate.getFullYear()
+    return `${day}-${month}-${year}`
+}
```

Comment:
Nice that you made your own function to do this, good separation of code so it can be re-used!